docs/en/modules/i2c.md

```diff
@@ -11,9 +11,11 @@ Setup I²C address and read/write mode for the next transfer.
 
 #### Parameters
 - `id` always 0
-- `device_addr` device address
+- `device_addr` device address (7-bit)<sup>[1](#7bit)</sup>
```

Comment:
I also think that `device_addr` can go without a reference, but propose to explain the construction of the resulting "8bit frame" in the command description above. Something like "`device_addr` is used for the upper 7 bits followed by the `direction` bit". This relates to the wording in [chapter 3.1.10 of the I2C spec](http://www.nxp.com/documents/user_manual/UM10204.pdf) and hopefully rules out misunderstanding.